pdm/utils.py

```diff
@@ -513,3 +516,15 @@ def pdm_scheme(base: str) -> dict[str, str]:
             "headers": "{base}/include",
         }
     return sysconfig.get_paths("pdm", vars={"base": base}, expand=True)
+
+
+def get_env_prefix() -> Optional[Path]:
```

Comment:
Make the name more obviously related to "venv"

Reply:
also `-> Path | None` to keep consistent

Reply:
Will do - I didn't notice the future import, so I wasn't sure about using the new union syntax!